service/src/main/resources/HTMLTemplate.html

```diff
@@ -867,6 +884,21 @@ <h3 class="heading heading--two">Child {{ counter }}</h3>
               {% endif %}
             </dd>
           </div>
+          <div class="data-list__row">
+            <dt class="data-list__title">Litigation capacity issues</dt>
+            <dd class="data-list__definition">
+              {% if child.value.litigationIssues is not empty %}
+                {% if child.value.litigationIssues == 'Yes' and
+                child.value.litigationIssuesReason is not empty %}
+                  <span class="u-block">{{ child.value.litigationIssuesReason }}</span>
```

Comment:
What we should do if somebody answered YES but gave no details? Shouldn't we say 'Yes' to still highlight there will be issues of unknown nature?

Reply:
The else block on 894 will do exactly that :)

Reply:
FYI - Will be making changes to the template in my next commit. Template should read 
Yes
{{ reason}}
---------
No
---------
Don't know